File: debug5.py
import sys
sys.stdout.reconfigure(encoding='utf-8')

filepath = r'C:\Users\wzgui\.openclaw-autoclaw\workspace\sportpredict\index.html'

with open(filepath, 'rb') as f:
    raw = f.read()
content = raw.decode('utf-8', errors='replace')

lines = content.split('\n')

# Check en block area - where do hero keys appear?
for i in range(1720, 1770):
    line = lines[i].rstrip('\r')
    if 'hero' in line.lower() or 'en:' in line.lower() or line.strip() == '},' :
        print(f"L{i+1}: {line[:200]}")

File: _debug5.js
const fs = require('fs');
const h = fs.readFileSync('C:/Users/wzgui/.openclaw-autoclaw/workspace/sportpredict/index.html', 'utf8');
const UQ = String.fromCharCode(65533) + '?';
const idx = h.indexOf(UQ);
if (idx >= 0) {
  const seg = h.substring(idx - 25, idx + 15);
  console.log('Context bytes:');
  for (let i = 0; i < seg.length; i++) {
    const ch = seg.charCodeAt(i);
    console.log('  ' + i + ': ' + seg[i] + ' (' + ch.toString(16) + ')');
  }
}
